packages/flutter/lib/src/material/floating_action_button_theme.dart

```diff
@@ -129,6 +132,11 @@ class FloatingActionButtonThemeData with Diagnosticable {
   /// The text style for an extended [FloatingActionButton]'s label.
   final TextStyle? extendedTextStyle;
 
+  /// {@macro flutter.material.RawMaterialButton.mouseCursor}
+  ///
+  /// If specified, overrides the default value of [FloatingActionButton.mouseCursor].
```

Comment:
Maybe `FloatingActionButton.mouseCursor`'s doc should be templated and macroed here instead of `RawMaterialButton.mouseCursor`?

Reply:
Other buttons are using the `flutter.material.RawMaterialButton.mouseCursor` doc template: https://github.com/flutter/flutter/pull/88470

Reply:
I was just referring to `FloatingActionButtonThemeData.mouseCursor`, as in theme data should macro its widget's member. But I don't see any other themes doing this, so this is fine ig.